src/python/pants/backend/python/packaging/pyoxidizer/config.py

```diff
@@ -11,12 +11,14 @@
 def make_exe():
     dist = default_python_distribution()
     policy = dist.make_python_packaging_policy()
+    policy.extension_module_filter = "no-copyleft"
```

Comment:
This is _very_ important if static linking gets enabled -- static linking is the trigger clause that makes the copyleft features of the GPL get switched on.

Reply:
This seems like a reasonable default for this flag, given the potential landmine. @sureshjoshi: are you ok with:
1. changing the default
2. exposing a flag on `pyoxidizer_binary` to override it?

Reply:
Sure, no problem with that. 

One heads up is that there is basically infinite configurability with this file, which is why I added this: https://www.pantsbuild.org/docs/reference-pyoxidizer_binary#codetemplatecode

Idea being, if the default "good enough" template wasn't enough, no one was perma-blocked - they could just create a custom `.bzlt` file and they're off to the races. I intentionally tried to use the PyOx defaults as much as possible for this default template... Principle of least surprise for incoming users.

So, I guess the question(s) is/are: what should the default template include, and is this use case is worth adding a specific flag or is it more of a "use the template" solution. 

I won't even pretend to have the answer to that question. Nor do I feel strongly about it. 🤷🏽 

Aside: Link to docs about this feature: https://pyoxidizer.readthedocs.io/en/stable/pyoxidizer_config_type_python_packaging_policy.html#starlark_pyoxidizer.PythonPackagingPolicy.extension_module_filter

Reply:
Thanks @sureshjoshi -- recognising that there is indeed a facility to override the entire template, I figured it was better to make the default template a safer default for people to use.

Reply:
Coolest of the beans. As mentioned, no strong opinions on that here.

I would suggest as we're using a non-default setting, it may be worth a note in the help